mono/mini/mini-runtime.c

```diff
@@ -3473,6 +3473,21 @@ mono_get_delegate_virtual_invoke_impl (MonoMethodSignature *sig, MonoMethod *met
 gboolean
 mini_parse_debug_option (const char *option)
 {
+	MonoDebugTailcallOptions * const tailcall = &mini_debug_options.tailcall;
+	switch (option [0]) {
+	case 't':
+		if (strcmp (option, "tailcall_break_compile") == 0)
```

Comment:
This is completely unnecessary and overengineered over what's actually needed. Even the previous approach of `#if 0` is much better than this.

Reply:
MONO_DEBUG is a thing. I didn't make it up.

You should be able to debug w/o edit and recompile/relink.

I find myself constantly changing the logging to be enabled under -v instead of -v -v, so I am not flooded with other output -- that even grep takes a while to get through -- and which runs tools that aren't usually on PATH on windows (objdump or somesuch).

Reply:
MONO_DEBUG should contain options which are somewhat commonly used. Its pretty common to recompile while debugging.